### PR TITLE
Pickup ammo

### PR DESCRIPTION
**Pull request by spiderbiggen**

### CODE REVIEW

**Comment by spiderbiggen** [comment]
I'm gonna add some tests for ammo pickup and then this can be merged I think

File: src/main/java/lasersharks/AmmoEatBehaviour.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * @author spide
+ *
+ */
+public class AmmoEatBehaviour implements EatBehaviour {
+
+  /**
+   * 
+   */
+  public AmmoEatBehaviour() {
+    // TODO Auto-generated constructor stub
```
Comment (Exclaminator) [inline src/main/java/lasersharks/AmmoEatBehaviour.java]: Please remove these auto bodies if there is nothing left to do.

Comment (spiderbiggen) [inline src/main/java/lasersharks/AmmoEatBehaviour.java]: oh oops i tought i had already done that

**Comment by SytzeAndr** [comment]
if it works, I'm oay with merging

**Comment by SytzeAndr** [comment]
it works, good enough

**Pull request merged**